templates/proxy.php.erb

```diff
@@ -0,0 +1,9 @@
+<?php
+  $allowed_urls = [
+<% @css_sheets.each do |sheet| -%>
```

Comment:
Idem for the select comment. Spaces between the operators would be appreciated too.